rest/changestest/changes_api_test.go

```diff
@@ -3619,12 +3619,16 @@ func TestChangesIncludeConflicts(t *testing.T) {
 func TestChangesLargeSequences(t *testing.T) {
 
 	if base.UnitTestUrlIsWalrus() {
-		t.Skip("TestChangesLargeSequences doesn't support walrus - needs to customize " + base.SyncSeqKey + " prior to db creation")
+		t.Skip("TestChangesLargeSequences doesn't support walrus - needs to customize " + base.DefaultMetadataKeys.SyncSeqKey() + " prior to db creation")
 	}
 	if !base.TestsDisableGSI() {
 		t.Skip("Requires N1QL due to CBG-361")
 
 	}
+
+	if base.TestsUseNamedCollections() {
+		t.Skip("Requires default collection to set " + base.DefaultMetadataKeys.SyncSeqKey())
+	}
```

Comment:
is it worthwhile making it possible to set large seqno on non default collections? for `RestTesterConfig` ?

Reply:
Probably worth it at some point, but I didn't think it was needed just for this test, as the test is mainly about json serialization working as intended in the REST API, and I thought this test running with the default collection would be sufficient coverage for regressions in that area.